### DISCUSSION TITLE
SQL Version 10.50.6220.0

### DISCUSSION THREAD

**Discussion by mentatpsi** [quartznet/quartznet#1060] [Q&A]
I'm having issues running your SQL Script on SQL Version 10.50.6220.0... the command you released for the SqlServer Script relies on minimum SQL Server 2016. What requirements do you have for Quartz 3.2.3.0? The command can be modified to run for SQL Server 10.50.6220 using old syntax. But wondering what other reliance on new Syntax there might be.

**Comment by lahma** [comment]
You should be able to use the older version of the script from earlier release or from Git history, Quartz doesn't depend on the syntax, just the schema.

You probably know you're out of support with your database version anyway.

**Reply by mentatpsi** [reply]
Just curious, which version would that be? I've found several versions in git history, but I would like to maintain the most recent up to date aside from the syntax difference.

**Reply by mentatpsi** [reply]
I can run the command 
git show 4dfe92c3093bc932785ce54f1dbc35283118c3be:./database/tables/tables_sqlServer.sql to get the prior commit version. I don't see the new Syntax. Are there any major additions that might create bugs I need to worry about?

Really appreciate the help.

**Reply by mentatpsi** [reply]
I was able to get it working by grabbing that file through the above command and redirecting it into a sql file. Did the trick.

**Reply by lahma** [reply]
Easiest is to see the history in GH UI: https://github.com/quartznet/quartznet/commits/master/database/tables/tables_sqlServer.sql then you can click a version, then it opens a diff and from there you can open the version of file in question.

![image](https://user-images.githubusercontent.com/171892/103402707-3e559980-4b56-11eb-8088-582eabc9c252.png)

After this version there's just a index change to improve index coverage and reduce redundancies (top commit).
